docs/decisions/0011-fhirserver.md

```diff
@@ -0,0 +1,130 @@
+
+Date: 2022-06-28
+
+## Status
+
+Accepted
+
+## Context and Problem Statement
+
+In healthcare, data is gathered from many different sources. Hospitals, prisons, schools, pharmacies, and more all gather healthcare data that state and local territories (STLTs) need to gather to report public health data accurately. In order to ingest data from these different reporting sites, there needed to be a way to standardize the data across all departments of health. 
+
+Previously, HL7 standards were the solution to this issue, but the problems of HL7 was that it was hard to use with modern internet architecture as well as being humanly unreadable. FHIR was created to solve both of these issues. 
+
+The FHIR (Fast Healthcare Interoperability Resources 1 ) standard defines how healthcare
+information can be exchanged between different computer systems regardless of how it is stored in those
+systems. It allows healthcare information, including clinical and administrative data, to be available
+securely to those who have a need to access it, and to those who have the right to do so for the benefit
+of a patient receiving care. The standards development organization HL7® (Health Level Seven®3) uses a
+collaborative approach to develop and upgrade FHIR.
+
+FHIR is a standardized data format that is used nationally to report health data. 
+
+A FHIR Server assists in bringing together data from various systems into one common, readable dataset. The FHIR Server itself does not do any analytical work, but aims to normalize public health data. In the case of this project, public health data will be reported from a variety of sources such as prisons, pharamacies, schools, and hospitals. A FHIR Server will ingest the records and become a single source of truth for a public health agency's data. 
+
+
+## Decision Drivers
+
+**Integration with Existing Systems** - The CDC has an existing cloud environment (Azure) so being able to work well with their existing system helps the development process. The team is working to get the first version into production so ease of maintainability and integration are important. 
+
+**Flexiblility** - While the team is developing the solution, flexibility is important as requirements are being discovered. 
+
+**Support for a broad and complex data model** -  The solution needs to capture all relevant data in a structured way. Healthcare data can be complex due to the different sources of healthcare data being reported differently. 
+
+**Analtyics** -  The platform needs to support epidemiologists' analytical tools.
+
+**History and Logging** - Healthcare data modifications, additions, and deletions need to be tracked over time for historical logging.
+
+**Time to market** - Working with a solution that accelerates our time to market 
+
+**Performance** - Performance in terms of availability, speed, reliability must be considered to develop a solution. 
+
+## Considered Options
+
+### FHIR Server vs. Custom Solution
```

Comment:
I'd suggest restructuring this a little bit.  Considered options include:
* FHIR Server
* RDBMS or Big Data platform (probably okay to combine these?)
* Data Streams via Message Queues

Reply:
Let me know what you think